features/topology-map/org.opennms.features.topology.app/src/main/java/org/opennms/features/topology/app/internal/TopologyUI.java

```diff
@@ -873,7 +873,6 @@ public LayoutManager getLayoutManager() {
     // may change during layout.
     private void loadUserSettings() {
         applyHistory(m_applicationContext.getUsername(), m_historyManager.getHistoryFragment(m_applicationContext.getUsername()));
-        m_graphContainer.redoLayout();
```

Comment:
If removing this, please ensure, that the topology is still loaded correctly. To do so, go to the Topology Map and get some state. Go to the right and click "Share View" (or similar). Copy the URL.
Now log out with the current user. Best option is to open a new private window in a seperate browser and paste in the URL. The same state should be visible. If that still works, this line can be removed.

Reply:
I went through your test case and it seems that this still works.
The call hierarchy is the following:
TopologyUI.init() -> loadUserSettings() this is the only time that this private method is called. Later in TopologyUI.init() we call m_graphContainer.redoLayout(); directly and thus the call m_graphContainer.redoLayout() in loadUserSettings() should be unnecessary?

Reply:
Yep, makes sense.